org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/converter/ZigBeeConverterSwitchLevel.java

```diff
@@ -48,23 +57,33 @@ public boolean initializeConverter() {
             return false;
         }
 
-        try {
-            CommandResult bindResponse = clusterLevelControl.bind().get();
```

Comment:
What is the reason that reporting is no longer setup for the level control cluster?

Reply:
Well spotted - that is indeed an error and that section should not have been removed.